src/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: white;
+}
+
+.card__image {
+  margin: 0 auto;
+  padding-bottom: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
You can use variables for repeated colors

Reply:
added to the code